problems/strings.js

```diff
@@ -3,23 +3,29 @@
  * @param {string} - str
  * @returns {string} - first char
  */
-function firstChar() {}
+const firstChar = (str) => {
+  return str.slice(0, 1);
```

Comment:
There's an easier way to do this. Think of how we find the first element of an array

Reply:
We try str[0] and it didn't pass the test!! That's why we use str.slice()